docs/introduction.rst

```diff
@@ -1,13 +1,26 @@
 RackHD Overview
 ===================
-In a Converged Infrastructure Platform (CIP) architecture, RackHD software provides hardware management and orchestration (M&O). It serves as an abstraction layer between other M&O layers and the underlying physical hardware. Developers can use the RackHD API to create a user interface that serves as single point of access for managing hardware services regardless of the specific hardware in place.
 
-Once RackHD is installed on the managed CIP platform, it has the ability to discover the existing hardware resources, catalog each component, and retrieve detailed telemetry information from each resource. The retrieved information can then be used to perform low-level hardware management functions for each resource, such as BIOS configuration, OS installation, and firmware management.
+RackHD serves as an abstraction layer between other M&O layers and the underlying physical
+hardware. Developers can use the RackHD API to create a user interface that serves as single point of access
+for managing hardware services regardless of the specific hardware in place.
 
-RackHD sits between the other M&O layers and the underlying physical hardware devices. User interfaces at the higher M&O layers can request hardware services from RackHD. RackHD handles the details of connecting to and managing the hardware devices.
+Once RackHD has the ability to discover the existing hardware
+resources, catalog each component, and retrieve detailed telemetry information from each resource. The retrieved
```

Comment:
This sentence is not grammatically correct and doesn't read well, since it starts with a conjunction ('Once') but never finishes it.